include/zephyr/linker/linker-defs.h

```diff
@@ -25,14 +25,7 @@
 #include <zephyr/sys/util.h>
 #include <offsets.h>
 
-/* We need to dummy out DT_NODE_HAS_STATUS when building the unittests.
- * Including devicetree.h would require generating dummy header files
- * to match what gen_defines creates, so it's easier to just dummy out
- * DT_NODE_HAS_STATUS.
- */
-#ifdef ZTEST_UNITTEST
-#define DT_NODE_HAS_STATUS(node, status) 0
```

Comment:
Curious, where is DT_NODE_HAS_STATUS(node, status) defined now / why is it no longer needed ?

Reply:
I'm also curious about that - could it possiby be documentation related?